### ISSUE TITLE
BusyBox setuidgid

### ISSUE THREAD

**Issue by mterron** [ncopa/su-exec#19]
How is this different from doing:
```
# exec setuidgid $USER command
```
or
```
# chpst -u $USER: -U $USER: command
```

**Comment by ploxiln** [comment]
Yeah this is does the same thing (with a bit more flexibility than setuidgid and a bit less than chpst). But many people won't have busybox already installed (or maybe their busybox package doesn't have these commands enabled) and this may be smaller and simpler than installing busybox.
